### ISSUE TITLE
Undefined variables, errors and other interesting 

### ISSUE THREAD

**Issue by Cenay** [soflyy/oxygen-bugs-and-features#1235]
*A clear and concise description of what the bug is.*
Receiving multiple PHP Notices about an undefined variable. While I was debugging another issue, this was appearing at the top of every page and broke the site's CSS (see attached image)

*Error Reported*
```
[09-Jun-2020 01:47:21 UTC] PHP Notice:  Undefined variable: oxy_nav_menu_styles_dropdowns in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3378
[09-Jun-2020 01:49:34 UTC] ... 7 more identical messages separated by seconds ...
[09-Jun-2020 01:50:07 UTC] PHP Notice:  Undefined variable: oxy_nav_menu_styles_dropdowns in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3378
```

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
*Yes/No*
**Yes, of course**

This isn't a failure causing issue and once debug is turned back off, it quietly goes back to reporting it to the error log and not displaying it on screen. However, this client pays for this storage (at Amazon) so having 1.8 GIG error log file (over the last 30 days) is adding to his monthly expenses. 

**Screenshots**
*If applicable, add screenshots to help explain your problem.*

https://www.screencast.com/t/3Xa9jfJrnnbE

**EDITED: To add this additional info**
```
[Mon Jun 08 23:28:59.510840 2020] [proxy_fcgi:error] [pid 2146:tid 139845175326464] [client 162.158.186.79:43418] AH01071: Got error 'PHP message: PHP Warning:  call_user_func_array() expects parameter 1 to be a valid callback, function 'disable_emojis' not found or invalid function name in /opt/bitnami/apps/wordpress/htdocs/wp-includes/class-wp-hook.php on line 287\nPHP message: PHP Warning:  strpos() expects parameter 1 to be string, array given in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3602\nPHP message: PHP Warning:  strpos() expects parameter 1 to be string, array given in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3606\nPHP message: PHP Warning:  strpos() expects parameter 1 to be string, array given in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3610\nPHP message: PHP Warning:  strpos() expects parameter 1 to be string, array given in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3614\nPHP message: PHP Warning:  strpos() expects parameter 1 to be string, array given in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3618\nPHP message: PHP Warning:  strpos() expects parameter 1 to be string, array given in /opt/bitnami/apps/wordpress/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php on line 3622\n'
```

This is also happening and I am not sure if it's related or not, so including it here. I did uncheck "disable" the Disable WP Emojis - and the error continues. 

**Desktop (please complete the following information):**
- OS: [e.g. iOS] Windows 10 Pro
 - Browser [e.g. chrome, safari] Firefox, Chrome Developers Edition FireFox
 - Oxygen Version [e.g. 2.2 Alpha] 3.3.1

**Additional context**
*Add any other context about the problem here.*

**Comment by Cenay** [comment]
Just had to delete a 225 meg error log due to this issue. 

Any progress? Suggestions for how to prevent? Timeline on when this might be addressed? This client is on Lightsail and pays for storage space, which is consistently being run way up due to the error log bloat with this issue.

**Comment by KittenCodes** [comment]
@Cenay Please see the list of bug fixes in 3.6 Beta - that should make a huge difference to the error log once the full version is released: https://oxygenbuilder.com/2020/09/28/oxygen-3-6-beta-1-now-available/.

Please note that the Beta isn't recommended for use on live sites.

**Comment by Cenay** [comment]
Thanks for the heads up @KittenCodes -- I look forward to the release of 3.6
